[2021-11-29 12:29:20,639] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_3 2020-01-27T00:00:00+00:00 [queued]>
[2021-11-29 12:29:20,717] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_3 2020-01-27T00:00:00+00:00 [queued]>
[2021-11-29 12:29:20,721] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-11-29 12:29:20,721] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-11-29 12:29:20,722] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-11-29 12:29:20,746] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_3> on 2020-01-27T00:00:00+00:00
[2021-11-29 12:29:20,767] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_3', '2020-01-27T00:00:00+00:00', '--job-id', '189', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmp7o9pd0dn', '--error-file', '/tmp/tmpv3ciqnid']
[2021-11-29 12:29:20,775] {standard_task_runner.py:52} INFO - Started process 135748 to run task
[2021-11-29 12:29:20,780] {standard_task_runner.py:77} INFO - Job 189: Subtask task_3
[2021-11-29 12:29:20,919] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_3 2020-01-27T00:00:00+00:00 [running]> on host airflowvm
[2021-11-29 12:29:21,189] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_3
AIRFLOW_CTX_EXECUTION_DATE=2020-01-27T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-27T00:00:00+00:00
[2021-11-29 12:29:21,190] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-11-29 12:29:21,192] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-11-29 12:29:21,212] {subprocess.py:75} INFO - Output:
[2021-11-29 12:29:24,257] {subprocess.py:83} INFO - Command exited with return code 0
[2021-11-29 12:29:24,514] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_3, execution_date=20200127T000000, start_date=20211129T122920, end_date=20211129T122924
[2021-11-29 12:29:24,917] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-11-29 12:29:24,941] {local_task_job.py:151} INFO - Task exited with return code 0
